rag-langchain/rag_app.py

```diff
@@ -0,0 +1,89 @@
+from langchain_openai import ChatOpenAI
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.runnables import RunnablePassthrough
+from langchain.embeddings.sentence_transformer import SentenceTransformerEmbeddings
+from langchain.callbacks import StreamingStdOutCallbackHandler
+from langchain.text_splitter import CharacterTextSplitter
+from langchain_community.document_loaders import TextLoader
+from langchain_community.vectorstores import Chroma
+
+from chromadb import HttpClient
+from chromadb.config import Settings
+import chromadb.utils.embedding_functions as embedding_functions
+
+
+import uuid
+import os
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-d", "--docs", default="data/fake_meeting.txt")
+parser.add_argument("-c", "--chunk_size", default=150)
+parser.add_argument("-e", "--embedding_model", default="BAAI/bge-base-en-v1.5")
```

Comment:
Do we want to make it customizable?

Reply:
Can you clarify? What would we want to customize here?